doc/md/privacy.md

```diff
@@ -374,23 +376,25 @@ func (TenantMixin) Edges() []ent.Edge {
 ```
 
 Now, we want to enforce that viewers can see only groups and users that are connected to the tenant they belong to.
-In this case, there's another type of privacy rule named `FilterRule`. This rule can help us to filters out entities that
-are not connected to the same tenant.
+In this case, we create another type of privacy rule named `FilterTenantRule`.  
+This rule can help us filter out entities that are not connected to the same tenant.
```

Comment:
A `FilterRule` rule is a **type** of privacy rule. The `FilterTenantRule` is an implementation. 

I hope this gives more context, and I'll be happy to improve this paragraph if it's not clear, but please mention the `FilterRule` type.

Reply:
@a8m check it now, please.